tests/data/test_us_equity_pricing.py

```diff
@@ -72,58 +77,57 @@
 TEST_QUERY_ASSETS = EQUITY_INFO.index
 
 
-class BcolzDailyBarTestCase(TestCase):
+class BcolzDailyBarTestCase(WithBcolzDailyBarReader, ZiplineTestCase):
+    BCOLZ_DAILY_BAR_START = SIM_PARAMS_START = TEST_CALENDAR_START
+    BCOLZ_DAILY_BAR_END = SIM_PARAMS_END = TEST_CALENDAR_STOP
 
     @classmethod
-    def setUpClass(cls):
-        all_trading_days = TradingEnvironment().trading_days
+    def make_equities_info(cls):
```

Comment:
I probably would have called this method `with_equity_info`, rather than `equities` info.

As a parallel, if you made a function returning a dataframe like:
```
    name  value
0  Scott     10
1    Joe      5
```
you would call it `make_user_info`, not `make_users_info`.